app/models/notification.rb

```diff
@@ -1,3 +1,24 @@
+# Super-class to hold information about all Notifications
 class Notification < ApplicationRecord
+  actable
+
   belongs_to :user
+
+  def custom_partial
+    specific.class.name.underscore
+  end
+
+  # delegate methods from the "subclasses" (which aren't really subclasses)
+  # to the specific instances
```

Comment:
This comment is a bit unclear to me. Could you clarify what these "subclasses" are?

Reply:
> I started to look through the changes and noticed that a lot of them introduced the lend request notification. From the description of the corresponding user story for this PR, the lend request notification should not be included here. Please clarify how we should deal with this. I think we should either check if everything regarding the lend request notification is included in this branch so we can combine the two features. Or otherwise, we could remove the lend request notification from this branch

This is due to us branching from PBRH+, since they had already implemented some of the more specific notification types.
We will have to wait with this merge until their changes have also been approved in #156 .